spec/stock_picker_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'stock_picker'
+describe 'stock_picker' do
```

Comment:
Are there additional edge cases that I can cover beyond the highest price on the first day and lowest price on the last?  I have added those tests, but just wondering if there are more.
Completed.